pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchpatch
+, meson
+, ninja
+, pkg-config
+, python3Packages
+, dbus
+, glslang
+, libglvnd
+, libXNVCtrl
+, mesa
+, vulkan-headers
+, vulkan-loader
+, xorg
+}:
+
+
+stdenv.mkDerivation rec {
+  pname = "mangohud${lib.optionalString stdenv.is32bit "_32"}";
+  version = "0.4.1";
```

Comment:
newest version is 0.6.1

Reply:
Versions newer than v4 won't work in Nixpkgs, see https://github.com/NixOS/nixpkgs/pull/95225#issuecomment-734643180